test/e2e/services.go

```diff
@@ -308,6 +412,24 @@ func newObservatoriumAPIService(
 		args = append(args, "--middleware.rate-limiter.grpc-address="+opts.ratelimiterAddr)
 	}
 
+	if opts.tracesWriteEndpoint != "" {
+		args = append(args, "--traces.write.endpoint="+opts.tracesWriteEndpoint)
```

Comment:
I think we could also update [usage](https://github.com/observatorium/api#usage) with those flags

Reply:
These were added in the previous PR, the one that added the functionality.

- https://github.com/observatorium/api/pull/199/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R78
- https://github.com/observatorium/api/pull/199/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R78